packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -805,6 +809,24 @@ export default defineComponent({
   }
 }
 
+.hoverable-quick-actions.files-table {
+  tr {
+    .resource-table-edit-name,
+    .resource-table-actions div:first-child {
+      visibility: hidden;
+    }
+  }
+}
+
+.files-table {
+  tr:hover {
+    .resource-table-edit-name,
+    .resource-table-actions div:first-child {
+      visibility: visible;
+    }
+  }
+}
```

Comment:
You can put the `hover` into the first one:

```suggestion
  tr:hover {
    .resource-table-edit-name,
    .resource-table-actions div:first-child {
      visibility: visible;
    }
  }
}
```